src/javaRosa.js

```diff
@@ -171,7 +171,14 @@ define([
             return Boolean(this.hasForm('default') || 
                            this.hasForm('long')    || 
                            this.hasForm('short'));
-        }
+        },
+        forEachLogicExpression: function (fn) {
+            var text = this.get(),
```

Comment:
Might want to add a test with an expression in non-default language, but not in the default.